CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Change Log
 
+
+## [1.5.0] - 2023-01-24
+
+### Added
+
+- Improve marker handling ([#528](https://github.com/python-poetry/poetry-core/pull/528),
+[#534](https://github.com/python-poetry/poetry-core/pull/534),
+[#530](https://github.com/python-poetry/poetry-core/pull/530),
+[#546](https://github.com/python-poetry/poetry-core/pull/546),
+[#547](https://github.com/python-poetry/poetry-core/pull/547)).
+- Allow an optional output directory for building ([#527](https://github.com/python-poetry/poetry-core/pull/527)).
+- Validate if dependencies referenced in `extras` are defined in the main dependency group ([#542](https://github.com/python-poetry/poetry-core/pull/542)).
+- Improve handling of generic constraints ([#515](https://github.com/python-poetry/poetry-core/pull/515)).
+
+### Changed
+
+- Deprecate the hash function of `FileDependency` ([#535](https://github.com/python-poetry/poetry-core/pull/535)).
+- Do not set `allows_preleases` implicitly anymore if the lower bound of a constraint is a pre-release ([#543](https://github.com/python-poetry/poetry-core/pull/543)).
+- Do not generate a `setup.py` by default anymore ([#318](https://github.com/python-poetry/poetry-core/pull/318)).
+- Remove the unused `platform` attribute from `Package` ([#548](https://github.com/python-poetry/poetry-core/pull/548)).
+- Deprecate the `pretty_version` parameter when creating a `Package` ([#549](https://github.com/python-poetry/poetry-core/pull/549)).
+- Allow path dependencies with a non-existent path ([#520](https://github.com/python-poetry/poetry-core/pull/520)).
```

Comment:
```suggestion
- Validate path dependencies during use instead of during construction ([#520](https://github.com/python-poetry/poetry-core/pull/520)).
```